frontend/src/metabase/home/components/EmbedMinimalHomepage/EmbedMinimalHomepage.tsx

```diff
@@ -27,53 +28,45 @@ export const EmbedMinimalHomepage = ({
     onDismiss();
   };
 
-  return (
-    <Stack>
-      <Text
-        fw="bold"
-        color="text-dark"
-      >{t`Get started with Embedding Metabase in your app`}</Text>
-      <Card px={40} py={32} maw={570}>
-        <Stack spacing="lg">
-          <Flex justify="space-between">
-            <Title size="h4">{t`As you expressed interest in Embedding, follow these steps to start`}</Title>
+  useEffect(() => {
+    // this card is only visible once
+    removeShowEmbedHomepageFlag();
+  }, []);
 
-            <Button
-              variant="white"
-              size="small"
-              color="text-dark"
-              p={0}
-              onClick={dismiss}
-            >
-              <Icon name="close" />
-            </Button>
-          </Flex>
+  if (!isAdmin) {
+    return null;
+  }
 
-          <NumberedList>
-            <li>{t`Enable and configure embedding in settings`}</li>
-            <li>
-              {t`Select or create a dashboard or question to do a static embed`}
-            </li>
-            <li>{t`Follow the quickstart to do an interactive embed`}</li>
-          </NumberedList>
+  return (
+    <Card px={40} py={32} maw={320}>
+      <Flex justify="space-between">
```

Comment:
I had to remove the comment for `eslint-disable-next-line no-literal-metabase-strings`, I guess we didn't have that rule in v49, the linter didn't let me commit otherwise

Reply:
Yeah, those rules aren't backported, because they're for new code only and mostly should be for features. Backports that fix bugs shouldn't have to be concern about the misuse of Metabase strings or doc links whatsoever.